src/progress-steps/types.js

```diff
@@ -40,20 +36,10 @@ export type StepPropsT = {
   isActive: boolean,
   /** Defines if the step is the last item displayed. */
   isLast: boolean,
-  overrides?: StepOverridesT,
+  overrides?: ProgressStepsOverridesT,
```

Comment:
It should still be `StepOverridesT` since `ProgressStepsOverridesT ` should only have `Root` in it.